README.md

```diff
@@ -34,146 +38,106 @@ Please help us by [Contributing](CONTRIBUTING.md) to the project.
 
-6. Cleanup the shared file system from the cluster (this **does** delete the data from the cluster)
+```
+$ kubectl cluster-info
+```
 
-   ```
-   ./rook filesystem delete --name testFS
-   ```
+If you see a url response, you are ready to go.
 
-#### Object Storage
-1. Create an object storage instance in the cluster
+#### Deploy Rook
 
-   ```bash
-   ./rook object create
-   ```
+Rook can be setup and deployed in Kubernetes by simply deploying the [rook-operator](demo/kubernetes/rook-operator.yaml) deployment manifest.
+You will this manifest and all our example manifests files in the [demo/kubernetes](demo/kubernetes) folder.
 
-2. Create an object storage user
+```
+$ cd demo/kubernetes
+$ kubectl create namespace rook
+$ kubectl create -f rook-operator.yaml
+```
 
-   ```bash
-   ./rook object user create rook-user "A rook rgw User"
-   ```
+Use `kubectl` to list pods in the rook namespace. You should be able to see the following: 
 
-3. Get the connection information for accessing object storage
+```
+$ kubectl -n rook get pod
+NAME                            READY     STATUS    RESTARTS   AGE
+mon0                            1/1       Running   0          1m
+mon1                            1/1       Running   0          1m
+mon2                            1/1       Running   0          1m
+osd-n1sm3                       1/1       Running   0          1m
+osd-pb0sh                       1/1       Running   0          1m
+osd-rth3q                       1/1       Running   0          1m
+rgw-1785797224-9xb4r            1/1       Running   0          1m
+rgw-1785797224-vbg8d            1/1       Running   0          1m
+rook-api-4184191414-l0wmw       1/1       Running   0          1m
+rook-operator-349747813-c3dmm   1/1       Running   0          1m
+```
 
-   ```bash
-   eval $(./rook object connection rook-user --format env-var)
-   ```
+#### Provision Storage
+Before Rook can start provisioning storage, a StorageClass needs to be created. This is used to specify the storage privisioner, parameters, admin secret and other information needed for Kubernetes to interoperate with Rook for provisioning persistent volumes.
+Rook already creates a default admin and demo user, whose secrets are already specified in the sample [rook-storageclass.yaml](demo/kubernetes/rook-storageclass.yaml).
 
-4. Use an S3 compatible client to create a bucket in the object store
+However, before we proceed, we need to specify the Ceph monitor endpoints. You can find them by running this line (you will need `jq`). This will generated a comma-separated list of monitor IPs and ports `6790` and place it into the `monitors` param of the `rook-storageclass.yaml`
 
-   ```bash
-   s3cmd mb --no-ssl --host=${AWS_ENDPOINT} --host-bucket=  s3://rookbucket
-   ```
+```
+$ export MONS=$(kubectl -n rook get pod mon0 mon1 mon2 -o json|jq .items[].status.podIP|tr -d "\""|sed -e 's/$/:6790/'|paste -s -d, -)
```

Comment:
With ZSH I had to single quote the jq argument like: `jq '.items[].status.podIP'` to avoid expansion/errors.  Should be safe for all shells that way.